src/components/Blogs/index.scss

```diff
@@ -0,0 +1,56 @@
+@import "../../style/variables";
+
+.blogsMainContainer {
+  text-align: center;
+  font-family: $mainFontFamily;
+
+  .container {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  .col {
+    width: 66.7rem;
+    margin-left: 1.5%;
+  }
+
+  .mr-sm-2 {
```

Comment:
if this is a bootstrap class name, you should add comment above it `// Bootstrap class`

Reply:
done